modules/privileged_access_manager/variables.tf

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "entitlement_id" {
+  type        = string
+  description = "The ID to use for this Entitlement. This will become the last part of the resource name. This value should be 4-63 characters. This value should be unique among all other Entitlements under the specified parent"
+}
+variable "location" {
+  type        = string
+  description = "The region of the Entitlement resource"
+  default     = "global"
+}
+
+variable "parent_id" {
+  type        = string
+  description = "The ID of organization, folder, or project to create the entitlement in"
+}
+
+variable "parent_type" {
+  type        = string
+  description = "Parent type. Can be organization, folder, or project to create the entitlement in"
+}
+
+variable "requester_justification" {
+  type        = bool
+  description = "If the requester is required to provide a justification"
+  default     = true
+}
+
+variable "entitlement_requesters" {
+  type        = list(string)
+  description = "Required List of users or groups who can request grants using this entitlement. Can be one or more of Google Account email, Google Group, Service account, or Google Workspace domain"
+}
+
+variable "entitlement_approvers" {
+  type        = list(string)
+  description = "Required List of users or groups who can request grants using this entitlement. Can be one or more of Google Account email, Google Group or Service account"
+}
```

Comment:
Does it make sense to collapse this (and the next two vars) into a map to reduce the no. of vars overall? This could be something like:

```
entitlement_users = map(list(string))

entitlement_users = {
   requesters = ["a", "b", "c"]
   approvers = ["x", "y", "z"]
}
```

Reply:
@g-awmalik due to the nature of these variables I prefer to keep it separate for sake of simplicity. Took care of rest of the comment. Thx